docs/getting-started/software-management/contributing-with-git.md

```diff
@@ -0,0 +1,164 @@
+# Contributing with Git
+
+## Prerequisites
+
+### Programming
+
+Before contributing, you should have an understanding of programming and of the language a program is written in.  For example, ISIS is written in c++, and ALE is written in Python.  If you don't know how to program, you might still be able to contribute by writing documentation or finding issues.
+
+### Installing Git
+
+***git*** is a version control system for managing an application's codebase, the code's history, and various versions and contributions to the code.
```

Comment:
Maybe put something at the top showing the overall git workflow as we use it. You can probably leverage the material theme's [mermaid support](https://squidfunk.github.io/mkdocs-material/reference/diagrams/) to make it?